.gitpod.yml

```
@@ -56,11 +56,15 @@ tasks:
       cp .gitpod/default.local.services.yml docroot/sites/local.services.yml &&
       blt blt:telemetry:disable --no-interaction &&
       blt settings &&
+      rm -rf .git &&
       drush si stanford_profile -v -y &&
       drush cr &&
       drush cim -y &&
       drush then $THEME_NAME &&
-      drush cset system.theme default $THEME_NAME -y
+      drush cset system.theme default $THEME_NAME -y &&
+      drush sqlq "UPDATE config_pages__su_site_dropdowns SET su_site_dropdowns_value = 1 WHERE bundle = 'stanford_basic_site_settings'" &&
+      drush php-eval 'use  \Drupal\node\Entity\Node; $node = Node::load(13); $node->setPublished(true); $node->save();' &&
+      drush php-eval '$menu_link_storage = \Drupal::entityTypeManager()->getStorage("menu_link_content"); $menu_link_storage->create(["title" => "Submenu link", "link" => ["uri" => "https://google.com"], "menu_name" => "main", "parent" => "menu_link_field:node_field_menulink_e18a741f-7d86-492a-a02f-7ca73989ca13_und", "expanded" => TRUE, "weight" => 0])->save();'
```

I'm not sure I understand the use case here, but if it helps development :+1:

---

This could even use the same logic as the node one:
```suggestion
      drush php-eval '\Drupal\menu_link_content\Entity\MenuLinkContent::create(["title" => "Submenu link", "link" => ["uri" => "https://google.com"], "menu_name" => "main", "parent" => "menu_link_field:node_field_menulink_e18a741f-7d86-492a-a02f-7ca73989ca13_und", "expanded" => TRUE, "weight" => 0])->save();'
```